<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameToolTab</class>
 <widget class="QWidget" name="RenameToolTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>264</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="folder_label">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="folder_edit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse_button">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filter_layout">
     <item>
      <widget class="QLabel" name="filter_label">
       <property name="text">
        <string>File Filter (regex):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filter_edit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>e.g. \.txt$ for text files only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="file_list_layout">
       <item>
        <widget class="QLabel" name="file_list_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Files in folder:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="file_list">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="operations_layout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="operations_label">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Rename Operations:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tab_widget">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="prefix_tab">
          <attribute name="title">
           <string>Prefix/Postfix</string>
          </attribute>
          <layout class="QFormLayout" name="prefix_form_layout">
           <item row="0" column="0">
            <widget class="QLabel" name="prefix_label">
             <property name="text">
              <string>Prefix:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="prefix_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="postfix_label">
             <property name="text">
              <string>Postfix:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="postfix_edit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="prefix_button">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Add Prefix/Postfix</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="regex_tab">
          <attribute name="title">
           <string>Replace</string>
          </attribute>
          <layout class="QVBoxLayout" name="regex_layout">
           <item>
            <layout class="QFormLayout" name="regex_form_layout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="pattern_label">
               <property name="text">
                <string>Find:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="pattern_edit">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>30</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Enter text to find or regex pattern like \(\d+\)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="replacement_label">
               <property name="text">
                <string>Replace With:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="replacement_edit">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>30</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>Enter replacement text or pattern like [\1]</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="transform_label">
               <property name="text">
                <string>Custom Transform:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="transform_edit">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>30</height>
                </size>
               </property>
               <property name="placeholderText">
                <string>e.g. f'prefix_{g1}_suffix' or 'g1.upper()'</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="button_layout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="regex_button">
               <property name="text">
                <string>Replace</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="custom_button">
               <property name="text">
                <string>Custom Transform</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="help_button">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(255, 244, 179);</string>
               </property>
               <property name="text">
                <string>Help</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
